### PR TITLE
Allow port to be changed with cli args

### PR DESCRIPTION
**Pull request by NervosaX**
This makes the cli self documenting with --help. 

I had to view the source code to see that you can use a PORT env variable to change the port... and then it turned out that it was hardcoded to port 3000 anyway (which just fails on my machine), so this also fixes that bug.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -31,6 +31,7 @@
     "styled-jsx": "^1.0.8"
   },
   "devDependencies": {
+    "argparse": "^1.0.9",
```
Comment (garrows) [inline package.json]: This will need to be a non-dev dependency

**Review by garrows — CHANGES_REQUESTED** [review]
Thanks heaps for this but can you make the change to dependencies?

File: package.json (hunk 1 above)
Comment (NervosaX) [reply package.json]: Ah crud, sorry about that.

Comment (NervosaX) [reply package.json]: @garrows Rebased, fixed that up.

**Comment by garrows** [comment]
Thanks. Merged and published.

**Pull request merged**